examples/ola-rdm.cpp

```diff
@@ -314,14 +342,45 @@ void RDMController::HandleResponse(
       // support queued messages.
       cout << "Empty STATUS_MESSAGES returned." << endl;
     }
-  } else if (response_status.response_type == ola::rdm::RDM_NACK_REASON) {
-    cout << "Request NACKed: " <<
-      ola::rdm::NackReasonToString(response_status.NackReason()) << endl;
+  } else if (response->ResponseType() == ola::rdm::RDM_NACK_REASON) {
```

Comment:
A NackReasonMessage method could wrap all of this, returning NackReasonToString or Invalid NACK reason size of " << response->ParamDataSize() << " expecting " << sizeof(nack_reason);